nest-forms-backend/src/forms/forms.helper.ts

```diff
@@ -1,106 +1,10 @@
 import { Injectable } from '@nestjs/common'
 import { Forms, FormState } from '@prisma/client'
 
-import { UserInfoResponse } from '../auth/decorators/user-info.decorator'
 import { EDITABLE_ERRORS } from '../utils/constants'
 
 @Injectable()
 export default class FormsHelper {
-  isFormAccessGranted(
-    form: Forms,
-    user: string | null,
-    ico: string | null,
-  ): boolean {
-    // if file is not owned by anyone, so we will show it
-    if (
-      form.userExternalId === null &&
-      form.ico === null &&
-      form.mainUri === null &&
-      form.actorUri === null
-    ) {
-      return true
-    }
-
-    // If file is owned by a company, the user must have the same ICO
-    if (form.ico !== null) {
-      return form.ico === ico
-    }
-
-    // if file is owned by the user through city account and not owned by eid uri
-    if (
-      form.userExternalId === user &&
-      form.mainUri === null &&
-      form.actorUri === null
-    ) {
-      return true
-    }
-
-    // if file is owned by someone else through city account and not owned by eid uri
-    if (
-      form.userExternalId !== user &&
-      form.userExternalId !== null &&
-      form.mainUri === null &&
-      form.actorUri === null
-    ) {
-      return false
-    }
-
-    // if file is owned by the user through eid uri and not owned by city account otherwise return false
-    return !(
-      form.mainUri !== null &&
-      form.actorUri !== null &&
-      form.userExternalId === null
-    )
-  }
-
-  userCanSendForm(
```

Comment:
These didn't really make sense. Now they all use the same function for access, no need to have separate check for send/send eID.